pages/common/for.md

```diff
@@ -0,0 +1,11 @@
+# for
+
+> Shell loop over parameters
+
+- Perform an action with different arguments.
+
+`for argument in 1 2 3; do {{cmd $argument}}; done`
+
+- Perform an action in every directory.
+
+`for d in *; do (cd $d; make test); done`
```

Comment:
Good catch. Perhaps a simpler command that makes it clear this is just an example command for demonstration, and not part of the `for` loop, would work better. I believe `echo` would be a fine choice for this.